tests/__snapshots__/index.test.ts.snap

```diff
@@ -408,6 +400,14 @@ exports[`Snapshot: punctuations 1`] = `
     12348,
     12351,
   ],
+  [
+    12448,
+    12448,
+  ],
+  [
+    12539,
+    12540,
+  ],
```

Comment:
Works as intended now.

```js
> String.fromCodePoint(12539)
'・'
> String.fromCodePoint(12540)
'ー'
> String.fromCodePoint(12448)
'゠'
```

Reply:
Yah, I made a new snapshot, easier to review.

Reply:
Looks good, right?

Reply:
![image](https://github.com/user-attachments/assets/3952f06f-71d0-4d1b-ac97-3e3b874a2fb6)

Reply:
I'll comment LGTM on the new diff.